cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -996,18 +996,27 @@ def alternative_titles_doc(self, key, value):
 @model.over("number_of_pages", "^300__")  # item
 def number_of_pages(self, key, value):
     """Translates number_of_pages fields."""
-    val = clean_val("a", value, str)
-    if is_excluded(val):
-        raise IgnoreKey("number_of_pages")
+    val_x = clean_val("x", value, str)
+    val_a = clean_val("a", value, str)
+    if val_x:
+        if val_x is "volume":
+            raise IgnoreKey("number_of_pages")
+        else:
```

Comment:
In the [issue](https://github.com/CERNDocumentServer/cds-ils/issues/299) one of the rules is:
"if `300__x` field present but empty migrate `300__a` => `physical_description`"
I checked some records and in some of them the `300__x` field is present but i's not empty, it contains `Phys.Desc.` (here a couple of examples: [record1](https://cds.cern.ch/record/280876/export/xm?ln=en) and [record2](https://cds.cern.ch/record/314343/export/xm?ln=en)).
That's why I am not checking if it is empty, is this ok?

Reply:
I checked and 300__x can only be [['Phys.Desc.'], ['Phys.desc']]